spec/lib/mastodon/cli/accounts_spec.rb

```diff
@@ -70,6 +70,25 @@ def account_from_options
         end
       end
 
+      context 'with min_age setting' do
+        let(:options) { { email: 'tootctl@example.com', confirmed: true } }
+
+        before do
+          Setting.min_age = 42
+        end
```

Comment:
If this is not rolled-backed automatically after each test, it might be good to do so manually in an `after` hook.